_data/stackx2021-cloud-conference-videos.yml

```
@@ -0,0 +1,80 @@
+- type: Video Types
+#   speakers: Speakers in the video. Split by ", "
+#     📢 Please make sure that the speakers name is in the same sequence as speakers title 📢 
+#   speakers_title: Speakers' title in the video. Split by "| "
+#   title: Video title
+#   id: Video Id
+#   Reference: https://docs.google.com/spreadsheets/d/1szkJAWTP0GZCNjZAxQobBAUsor_4URaqheqH70r5BZI/edit#gid=0
+
+- type: Keynote-VOD
+  speakers: 'Dr Janil Puthucheary'
+  speakers_title: 'Senior Minister of State at Ministry of Communications & Information and Ministry of Health, and Minister-in-Charge of GovTech'
+  title: 'Welcome Remarks'
+  id: '3Wztk782Sgc'
+- type: Keynote - VOD
+  speakers: 'Tom Soderstrom'
+  speakers_title: 'Director of Chief Technologists, Worldwide Public Sector, Amazon Web Services'
+  title: 'Impact and Lessons Learned from NASA/Jet Propulsion Laboratory's Cloud Computing Journey'
```

```suggestion
  title: "Impact and Lessons Learned from NASA/Jet Propulsion Laboratory's Cloud Computing Journey"
```
The single quotes should be replaced by the double quotes as there is another single quote in Laboratory**'s** causing the yaml to be invalid.

You can check the validity of the yaml with this tool https://onlineyamltools.com/validate-yaml